### DISCUSSION TITLE
Which one is the minimum of "cache_seconds"? 2 or 4 hrs?

### DISCUSSION THREAD

**Discussion by SherkeyXD** [anuraghazra/github-readme-stats#2411] [Q&A] [answered]
At first the documentation showed that it's 2 hours (7200 seconds)
> cache_seconds - set the cache header manually (min: **7200**, max: 86400). Default: 14400 seconds (4 hours).

While down below, it said that that should be 4 hours

>Also, note that the cache is clamped to a minimum of **4 hours** and a maximum of 24 hours.

Which one is right?

**Comment by Zo-Bro-23** [comment, marked as answer]
https://github.com/anuraghazra/github-readme-stats/blob/06a2a78cfcb1cd29c3b8b46d3899ffac10ab14b8/api/index.js#L58-L62

The code uses four hours; we should probably update the README (@rickstaa?).

**Reply by rickstaa** [reply]
@Zo-Bro-23 You are right. It was recently increased to 4 hours to deal with the Public Vercel instance downtime. We still need to add it to the README since we hoped we would be able to solve the problem. However, the popularity of this repository is increasing day by day, so, at the current time, I don't think we can revert to 2 hours until the proposed GitHub action version is implemented (see #2179). Feel free to create a PR for updating the cache.

**Reply by Zo-Bro-23** [reply]
Random PATs don't seem to work? I would assume that creating 10 or 20 PATs would solve the issue at least until the repo becomes more popular.

**Reply by Zo-Bro-23** [reply]
> Feel free to create a PR for updating the cache.

Done (#2442).
